tests/pages/home_page.py

```
@@ -1,18 +1,17 @@
-from asyncio import timeout
-
 from selenium.webdriver.support import expected_conditions as EC
-from selenium.webdriver import Keys
-from selenium.webdriver.common.by import By
 from selenium.webdriver.support.wait import WebDriverWait
+from tests.utilities.locators import HomePageLocators, CommonLocators
+
 from tests.pages.base_page import BasePage
 from tests.utilities.locators import HomePageLocators
```

Ruff (0.15.0)

[error] Redefinition of unused `HomePageLocators` from line 3: `HomePageLocators` redefined here

Remove definition: `HomePageLocators`

(F811)